.github/workflows/database.yml

```diff
@@ -0,0 +1,42 @@
+name: Database CI
+on: push
+jobs:
+  migrations:
```

Comment:
The next section is the `jobs` section. A workflow is made up of some number of jobs, which are basically the individual checks you see on each PR. Each job does one task, and either succeeds or fails (or gets skipped).

This workflow (database.yml) has one job, which is to run the migrations. The other workflow (javascript.yml) has two jobs, to run `lint` and to run `test` separately so we can see one check for each on the PR.